src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/BddSteps.kt

```diff
@@ -202,6 +268,23 @@ class BddSteps : En {
       val calculation: TierCalculationEntity = getTier()
       assertThat(calculation.data.protect.points).isEqualTo(points.toInt())
     }
+
+    Then("a Change level of {string} is returned for {string} points") { changeLevel: String, points: String ->
+      await untilCallTo {
+        getNumberOfMessagesCurrentlyOnQueue(
+          calculationCompleteClient,
+          calculationCompleteUrl
+        )
+      } matches { it == 1 }
+      val message = calculationCompleteClient.receiveMessage(calculationCompleteUrl)
+      val sqsMessage: SQSMessage = gson.fromJson(message.messages[0].body, SQSMessage::class.java)
+      val changeEvent: TierChangeEvent = gson.fromJson(sqsMessage.Message, TierChangeEvent::class.java)
+
+      val calculation: TierCalculationEntity? = tierCalculationRepository.findByCrnAndUuid("X12345", changeEvent.calculationId)
+      assertThat(calculation?.data?.change?.points).isEqualTo(Integer.valueOf(points))
+      assertThat(calculation?.data?.change?.tier?.value).isEqualTo(Integer.valueOf(changeLevel))
+    }
+
```

Comment:
Can largely be replaced by getTier, which returns a non-nullable TierCalculationEntity